lib/ethon/easy/informations.rb

```diff
@@ -46,6 +46,12 @@ module Informations
         # start until the name resolution was completed.
         :namelookup_time => :double,
 
+        # Return the time, in seconds, it took for all redirection steps
+        # include name lookup, connect, pretransfer and transfer before the
+        # final transaction was started. time_redirect shows the complete
```

Comment:
This is the raw curl documentation I copied to be consistent with other comments in this file.
The curl field is named `time_redirect` and then ethon reverses it. But you can change it if you want :)